src/drivers/uavcan/uavcan_main.cpp

```diff
@@ -905,6 +911,7 @@ int UavcanNode::run()
 				// but this driver could well serve multiple groups.
 				unsigned num_outputs_max = 8;
 
+				_mixers->set_thrust_factor(_thr_mdl_factor);
```

Comment:
It's minor, but rather than storing `_thr_mdl_factor` and making the call every single actuator update you could just do it once directly in the param update if _mixers isn't null.

Reply:
@dagar I was about to change it but @bkueng correctly mentioned that this is a problem when a new mixer is appended when the module is already running.

Reply:
Yes I suppose that's true, although we tend to get away with things like that because there's a storm of parameter updates early on. We can come back to the idea later with some common structure for these output modules.